anet.yml

```diff
@@ -214,29 +214,71 @@ dictionary:
         style:
           height: 400px
       topLevel:
-         shortLabel: Objective
-         shortName:
-           label: Objective name
-           placeholder: Enter an objective name, example....
-         longLabel: Objectives
-         longName:
-           label: Objective description
-           placeholder: Enter an objective description, example ....
-           componentClass: textarea
-           style:
-             height: 400px
+        shortLabel: Objective
+        shortName:
+          label: Objective name
+          placeholder: Enter an objective name, example....
+        longLabel: Objectives
+        longName:
+          label: Objective description
+          placeholder: Enter an objective description, example ....
```

Comment:
The phrasing "objective name/description" might be confusing to the user. Maybe change to something like:
```
          label: Name of this Objective
          placeholder: Enter a Name for this Objective name, example....
        longLabel: Objectives
        longName:
          label: Description of this Objective
          placeholder: Enter a description for this Objective, example ....

```

Reply:
done